File: bench.cs
using System;

public class bench
{
    public static int RunCode(int number)
    {
        if (number == 1)
            return 1;
        else {
            int num1 = number;
            int num2 = number;
            for (int i = 0; i < number * 1000; i++) {
                num2 *= number;
            }
            return num2 * RunCode(num1 - 1);
        }
    }
    public static void Main(String[] args)
    {
        RunCode(10);
    }
}

File: bench.cpp
double run_code(double number)
{
    if (number == 1)
        return 1;
    else {
        double num1 = number;
        double num2 = number;
        for (double i = 0; i < number * 1000; i++) {
            num2 *= number;
        }
        return num2 * run_code(num1 - 1);
    }
}

int main(int argc, char** argv)
{
    run_code(10);
}
